src/test/regress/sql/multi_shard_update_delete.sql

```diff
@@ -0,0 +1,271 @@
+--
+-- multi shard update delete
+-- this file is intended to test multi shard update/delete queries
+--
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1440000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1440000;
+
+SET citus.shard_replication_factor to 1;
+SET citus.multi_shard_modify_executor to 'parallel';
+
+CREATE TABLE users_copy_table (like users_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_distributed_table('users_copy_table', 'user_id');
+INSERT INTO users_copy_table SELECT * FROM users_table;
+
+CREATE TABLE events_copy_table (like events_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_distributed_table('events_copy_table', 'user_id');
+INSERT INTO events_copy_table SELECT * FROM events_table;
+
+CREATE TABLE events_reference_copy_table (like events_reference_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_reference_table('events_reference_copy_table');
+INSERT INTO events_reference_copy_table SELECT * FROM events_reference_table;
+
+CREATE TABLE users_reference_copy_table (like users_reference_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_reference_table('users_reference_copy_table');
+INSERT INTO users_reference_copy_table SELECT * FROM users_reference_table;
+
+-- Run multi shard updates and deletes without transaction on hash distributed tables
+SELECT COUNT(*) FROM users_copy_table;
+UPDATE users_copy_table SET value_1 = 1;
+SELECT SUM(value_1) FROM users_copy_table;
+
+SELECT COUNT(*) FROM users_copy_table WHERE user_id = 20 or user_id = 8 or user_id = 60;
+SELECT SUM(value_2) FROM users_copy_table WHERE user_id = 20 or user_id = 8 or user_id = 60;
+UPDATE users_copy_table SET value_2 = value_2 + 1 WHERE user_id = 20 or user_id = 8 or user_id = 60;
+SELECT SUM(value_2) FROM users_copy_table WHERE user_id = 20 or user_id = 8 or user_id = 60;
```

Comment:
These queries are dizzying and the contents of the table kind of depends on other tests. Could we do something that will be simpler for future readers to follow? Maybe with some manual data.

Reply:
Done